css/main.css

```diff
@@ -27,8 +27,8 @@
   background-color: none;
   transition: background-color 200ms linear;
   display: flex;
-  padding-top: 0.01rem;
-  padding-bottom: 0.01rem; }
+  padding-top: 1%;
+  padding-bottom: 1%; }
```

Comment:
i just realized this left a white bar -- thoughts on how to make the orb feel less squished from top & bottom without messing this up?

Reply:
oh lol dont worry abt that whitespace - thats bc i have some hardcoded things for the "About" section which i can always fix